stdlib/ballerina-http/src/main/ballerina/http/http_push_promise.bal

```diff
@@ -38,5 +43,8 @@ public type PushPromise object {
 
     @Description {value:"Removes all transport headers from the Push Promise"}
     public native function removeAllHeaders ();
+
+    @Description {value:"Gets all transport header names from the Push Promise."}
+    public native function getHeaderNames () returns (string[]);
```

Comment:
Shall we have doc comment for return type?

Reply:
sure, forgot to add it :(